cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -602,4 +603,35 @@ protected Properties getDefaultLogConfig() {
         }
         return properties;
     }
+
+    public ServiceAccount generateInitContainerServiceAccount() {
+        return new ServiceAccountBuilder()
+                .withNewMetadata()
+                    .withName(getInitContainerServiceAccountName(name))
+                    .addToLabels("app", "strimzi")
+                    .withNamespace(namespace)
+                .endMetadata()
+            .build();
+    }
+
+    public static String getInitContainerServiceAccountName(String name) {
+        return name + "-kafka-init";
+    }
+
+    public static String getInitContainerClusterRoleBindingName(String name) {
+        return "strimzi-" + name + "-kafka-init";
```

Comment:
Since this is the name for a clusterrolebinding it's globally scoped. It ought to be possible to deploy two instances of the CO in the same cluster, so perhaps this needs to include something about the namespace in which the CO is deployed in order to avoid collision?